database/client.go

```
@@ -0,0 +1,26 @@
+
+func Connect(connectionString string) {
+	Instance, err = gorm.Open(mysql.Open(connectionString), &gorm.Config{})
+	if err != nil {
+		log.Fatal(err)
+		panic("Cannot connect to DB")
+	}
+	log.Println("Connected to Database...")
+}
+
+func Migrate() {
+	Instance.AutoMigrate(&entities.Product{})
```

golangci-lint (1.64.8)

Error return value of `Instance.AutoMigrate` is not checked

(errcheck)